com.ranfdev.Notify.json

```diff
@@ -0,0 +1,76 @@
+{
+    "id": "com.ranfdev.Notify",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "command": "notify",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=RUST_LOG=notify=info,ntfy_daemon=info",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--talk-name=org.freedesktop.Notifications"
```

Comment:
is this necessary ? Gtk does the right thing.

Reply:
Probably it's not necessary, but I'm already having a really hard time pushing notifications through the Notification portal, because of https://github.com/flatpak/xdg-dbus-proxy/issues/46. 

I prefer to keep "--talk-name=org.freedesktop.Notifications" just to have less possible things causing problems while sending a notification.